server/request_object.cc

```diff
@@ -90,6 +90,20 @@ void s3_client_read_timeout_cb(evutil_socket_t fd, short event, void* arg) {
   request->trigger_client_read_timeout_callback();
 }
 
+void s3_response_delay_timeout_cb(evutil_socket_t fd, short event, void* arg) {
+  s3_log(S3_LOG_DEBUG, "", "%s Entry\n", __func__);
+
+  S3Action* action_obj = static_cast<S3Action*>(arg);
+
```

Comment:
you can have the logging Entry with request id here

Reply:
We don't have request id here. I'm passing action object in 'arg', not request object.

Reply:
S3Action has access to request Object(request_id)